org.eclipse.winery.topologymodeler.ui/src/app/models/ttopology-template.ts

```diff
@@ -118,16 +118,25 @@ export class TNodeTemplate extends AbstractTTemplate {
     }
 }
 
-/**
- * This is the datamodel for the Entity Types
- */
-export class EntityType {
+export class Entity {
     constructor(public id: string,
                 public qName: string,
                 public name: string,
-                public namespace: string,
+                public namespace: string) {
+    }
+}
+
+/**
+ * This is the datamodel for the Entity Types
```

Comment:
I would remove this comment, because it is the same as the code says.

Java-by-Comparison: "Remove Superfluous Comments"